float64 west_to_east_velocity
float64 south_to_north_velocity